### ISSUE TITLE
Error: Object.fromEntries() requires a single iterable argument on old browsers (ie11 specifically)

### ISSUE THREAD

**Issue by thenano** [garronej/tss-react#21]
First of all I'd like to express my gratitude on the great work done on this, it's made my migration to MUI 5 actually possible. Thanks!
Second, I know that a lot of current libraries have dropped ie11 support, and even MUIv5 states that there is only partial support for it.
I just wanted to report the issue in the hopes that there is an easy solution where we can allow our users to have some sort of experience with ie11, even if not ideal, as opposed to have to show them an error page and not being able to style anything.

Thanks

**Comment by garronej** [comment]
Thank you for expressing appreciation for the work that have been put into it! <3  
I will try to make it compatible with ie11. I am quite surprised though because I am already using [a polyfill for Object.fromEntries](https://github.com/garronej/tss-react/blob/fe31d3ba3c2caac42c19dc440740e474a681986e/src/tools/Object.fromEntries.ts)

**Comment by thenano** [comment]
Awesome, thanks you so much

**Comment by garronej** [comment]
I started to have a look. But I won't be able to finish tonight. Maybe tomorrow.

**Comment by garronej** [comment]
I got [the test SPA app](https://github.com/garronej/tss-react/tree/main/src/test/spa) to work on ie11. It uses mui v5.  I couldn't reproduce the error you mentioned. 

![image](https://user-images.githubusercontent.com/6702424/133341057-a44b9e69-7b14-4081-a618-7dbaff7a5486.png)

Anyway I will try to fix my polyfill of `Object.fromEntry` maybe it will fix it for you.

**Comment by garronej** [comment]
If you use create-react-app maybe it you are missing somme polyfills? 
See [here](https://github.com/garronej/tss-react/blob/d7e2f62d6195ec8292fbaae371be080ee1076a4b/src/test/spa/src/index.tsx#L1-L2) and [here](https://github.com/garronej/tss-react/blob/399b359393465cb9ca018595d2e705adca83f48a/src/test/spa/package.json#L32-L37)

**Comment by garronej** [comment]
Anyway I updated my polyfill in [the last version](https://github.com/garronej/tss-react/releases/tag/v0.9.1) maybe it will work now.

**Comment by thenano** [comment]
@garronej thanks so much for all your help with this. I'll test the new version and report back

**Comment by thenano** [comment]
Hi @garronej I updated the package and was still getting the same error, but like you said, I was missing some polyfills. I deliberately added a polyfill for Object.fromEntries and it's all good now. Just as reference, I don't use the react-app-polyfill package (it's so large! I use polyfill.io instead) and I'm somewhat ejected from CRA (I use react-app-rewired reluctantly for other reasons that I wish didn't exist).
I added the polyfill for Object.fromEntries from polyfill.io and that solved it.

Thanks again for all your help and such a speedy response to this issue, I'll close it now.
